src/hardware/vga_s3.cpp

```diff
@@ -27,9 +27,16 @@
 #include "../ints/int10.h"
 #include "inout.h"
 #include "mem.h"
+#include "pci_bus.h"
 #include "support.h"
 #include "vga.h"
 
+#if defined(PCI_FUNCTIONALITY_ENABLED)
+
+void PCI_AddSVGAS3_Device();
```

Comment:
Weird function name... What's up with that underscore in the middle?
Maybe too late, but I prefer this style: `PCI_AddSvgaS3Device`

Reply:
Yeah, I don't like this name either, preserved it only to keep the name from the original patch. It used to be an extern function defined in `pci_bus.h` (hence `PCI_` prefix / "namespace"), but then it got moved here in the latest version of the patch.